python/ccf/read_ledger.py

```diff
@@ -123,15 +130,15 @@ def dump_entry(entry, table_filter, tables_format_rules):
 def run(
     paths,
     is_snapshot=False,
-    tables=".*",
+    tables=None,
     uncommitted=False,
     insecure_skip_verification=False,
     tables_format_rules=None,
     digests_only=None,
 ):
 
     # Extend and compile rules
-    table_filter = re.compile(tables)
+    table_filter = re.compile(tables) if tables is not None else None
```

Comment:
```suggestion
    table_filter = re.compile(tables or ".*")
```

but that's clearly opinionated nitpicking